src/sass/components/_button.scss

```diff
@@ -33,4 +33,11 @@
     background-color: #2a2a2a;
     color: #ffffff;
   }
+  &.active {
+    background-color: $primary;
+    border-color: $primary;
+    color: #ffffff;
```

Comment:
Raczej zastosowałbym format trójkowy, ale to na przyszłość. Jeśli nasze pary się powtarzają czyli mamy `ffeedd` to możemy zrobić `fed` i to będzie znaczyło to samo, ale to zapamiętaj sobie na przyszłość 👍